grails-app/services/au/org/ala/profile/hub/ProfileService.groovy

```diff
@@ -66,6 +66,10 @@ class ProfileService {
         webService.get("${grailsApplication.config.lists.base.url}/ws/speciesListItems/${drid}?includeKVP=true")?.resp
     }
 
+    def getPublications(String pubId) {
+        webService.get("${grailsApplication.config.profile.service.url}/publication/${enc(pubId)}")?.resp
```

Comment:
If you return the same object that WebService.get returns, you can use the "handle" method on the base controller, which will deal with errors as well as the response.